ci_microshift.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -exuo pipefail
+
+sudo yum install -y podman make golang rsync
```

Comment:
podman is also installed by the `microshift.sh` script, might not be needed here? https://github.com/crc-org/snc/blob/cfe2c7dc246fd8efb769a6cd0bdbb80d7280d3b0/microshift.sh#L57

Reply:
also `rsync` is not being used by the script?